src/Components/App.js

```diff
@@ -0,0 +1,23 @@
+import { useQuery, gql } from '@apollo/client';
+import './NavBar';
+import NavBar from './NavBar';
+import styled from 'styled-components';
+
+
+function App() {
+  return (
+    <div>
+      <NavBar/>
+      <Img src={require ("../assets/myrlene-numa-SnITZTTeJVE-unsplash.jpg")}/>
+      <div>
+      Placeholder for Craft Container
+      </div>
+    </div>
+  );
+}
+
+export default App;
+
+const Img = styled.img`
+width: 100vw;
+`;
```

Comment:
Nice implementation of styled components! I look forward to seeing how this makes our codebase more streamlined as our application expands in scope.